pkg/apis/velero/v2alpha1/data_upload_types.go

```diff
@@ -96,6 +96,16 @@ const (
 	DataUploadPhaseFailed     DataUploadPhase = "Failed"
 )
 
+// NodeOS represents OS of a node.
+// +kubebuilder:validation:Enum=auto;linux;windows
```

Comment:
```suggestion
// +kubebuilder:validation:Enum=auto;linux;windows
// +kubebuilder:default=linux
```
Instead of patching to linux on empty in our code, we could force a default in CRD. WDYT?

Reply:
Might be less of a concern since this is not meant to be frequently created manually.

Reply:
This CRD is used by Velero internally, and in every occasion, Velero will set a valid value. So to make the logics more clear, we don't need to set a default value.